lib/connection/result/result.js

```diff
@@ -2,14 +2,14 @@
  * Copyright (c) 2015-2021 Snowflake Computing Inc. All rights reserved.
  */
 
-var EventEmitter = require('events').EventEmitter;
-var Util = require('../../util');
-var Errors = require('../../errors');
-var Chunk = require('./chunk');
-var ResultStream = require('./result_stream');
-var ChunkCache = require('./chunk_cache');
-var Column = require('./column');
-var StatementType = require('./statement_type');
+const EventEmitter = require('events').EventEmitter;
```

Comment:
are we planning to replace var with const/let now? If so, better to make it in separated PR to ensure all var get replaced an no mixed stuff left.

Reply:
Okay, I reverted them, and pushed the commit again.